code-challenges.test.js

```diff
@@ -15,45 +15,127 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
+//var num1 = 15
 // Expected output: "15 is divisible by three"
-var num2 = 0
+//var num2 = 0
 // Expected output: "0 is divisible by three"
-var num3 = -7
+//var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+//1. "describe" the name of the function in this case I'll name it "divisibleBy"
+//2. "it" is the next step. I'll describe the "divisibleBy" function does. I'll explain it in a string or plain words/sentenses like "takes a number as an argument and decides if the number is evenly divisble by three or not"
+//3. "expect and toEqual" I'll call on the divisibleBy() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+// I'll add the expect 18 and I'll add the toEqual 18 is divisible by three
+// add more expect 10 and I'll add the toEqual 10 is not divisible by three
+//4. I'll run the test on yarn jest -- expect the test to fail (get the red)
+//5. I'll make the function "divisibleBy" using method (num % 3 === 0) return num is divisible by three
+//6. Run the test -- get the green
+//7. Refactor the code if needed (make it more prettier)
 
+describe("divisibleBy", () => {
+  it("takes a number and decides if the number is evenly divisble by three or not based on the input", () => {
+    expect(divisibleBy(18)).toEqual("18 is evenly divisible by 3")
+    expect(divisibleBy(10)).toEqual("10 is not evenly divisible by 3")
+  })
+})
+// // Got the red ---
```

Comment:
Your test syntax looks good but the goal is to use the test variables and to get the exact outcome.